### PR TITLE
Port RTN and GPTQ pt examples to 3.x 

### PR DESCRIPTION
**Pull request by Kaihui-intel**
## Type of Change

feature

## Description

Port RTN and GPTQ pt examples to 3.x : gpt-j-6b and opt-125m

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

None

### CODE REVIEW

File: examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+datasets >= 1.1.3
+torch >= 1.10
+transformers
+pytest
+wandb
+einops
+neural-compressor
+intel-extension-for-transformers
+git+https://github.com/EleutherAI/lm-evaluation-harness.git@83dbfbf6070324f3e5872f63e49d49ff7ef4c9b3
+git+https://github.com/huggingface/peft.git@6c44096c7b8d55a2ecf24be9bc68393467e1584a
```
Comment (chensuyue) [inline examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/requirements.txt:13]: To match ITREX 1.3, `lm-eval` should be update into `git+https://github.com/EleutherAI/lm-evaluation-harness.git@cc9778fbe4fa1a709be2abed9deb6180fd40e7e2`.   
And peft can be the released one, so just use `peft` without commit.

File: examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    --approach weight_only \
+    --woq_bits 4 \
+    --woq_group_size 128 \
+    --woq_scheme asym  \
+    --woq_algo GPTQ \
+    --woq_enable_mse_search \
+    --output_dir "saved_results"
+    --accuracy
+```
+**Notes**: Weight-only quantization based on fake quantization is previewly supported and supports RTN, GPTQ[1], AWQ[2], TEQ algorithms. For more details, please refer to [link](https://github.com/intel/neural-compressor/blob/master/docs/source/quantization_weight_only.md)
```
Comment (chensuyue) [inline examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/README.md:55]: Use I know, for 3.x we only support RTN and GPTQ, right?

File: examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/run_clm_no_trainer.py [hunk 3]
```diff
@@ -0,0 +1,451 @@
...
+
+        conf = PostTrainingQuantConfig(
+            approach=args.approach,
+            op_type_dict=op_type_dict,
+            op_name_dict=op_name_dict,
+            recipes=recipes,
+        )
+
+        # for test on various models, keep the code of directly call gptq_quantize
+        if args.gptq_debug:
```
Comment (chensuyue) [inline examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/run_clm_no_trainer.py:255]: This branch has some issues, you can edit the code based on https://github.com/intel/neural-compressor/pull/1475/files#diff-6bf78d064d7b7861d36fc73e88a0e1c4c75c21b44bc598abbbdd4cc508a70981R294

**Comment by Kaihui-intel** [comment]
This PR will close, ports to https://github.com/intel/neural-compressor/pull/1498

File: examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+Step-by-Step
```
Comment (chensuyue) [inline examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/README.md:1]: Please add `run_quant.sh` and `run_benchmark.sh` for extension test.

**Pull request closed without merging**